docs/visual-testing/integrations/cypress.md

```diff
@@ -0,0 +1,200 @@
+```
+
+### Step 5: Run the test
+
+Upon executing your tests for the first time under this step, a visual baseline is automatically created in our system. This baseline serves as the standard for all subsequent Cypress tests. As new tests are run, they are compared to this original baseline, with any deviations highlighted to signal visual changes. These comparisons are integral for detecting any unintended visual modifications early in your development cycle. All test builds, including the initial baseline and subsequent runs, can be monitored and managed through the Sauce Labs platform at [Sauce Labs Visual Builds](https://app.saucelabs.com/visual/builds).
+
+Remember, the baseline is established during the initial run, and any subsequent visual differences detected will be marked for review.
+
+## Advanced usage
+
+### Build name
+
+Sauce Visual for Cypress plugin extends Cypress configuration, allowing to define the context, thus acting on which baselines new snapshots will be compared to. ([More info on baseline matching](/visual-testing/#baseline-matching))
+
+Options:
+
+- `region`: Sauce Labs Region where the new build will be created (default: `us-west-1`)
+- `buildName`: Name of the build (default: `Cypress Visual Testing`)
+- `project`: Name of the project (default: `None`)
+- `branch`: Name of branch (default: `None`)
+
+They need to be set through the `saucelabs` attribute of `e2e` configuration.
+
+Example:
+
+```javascript
+export default defineConfig({
+  e2e: {
+    [...]
+    saucelabs: {
+      region: 'eu-central-1',
+      buildName: 'SauceDemo - Cypress',
+      project: 'Cypress examples',
+      branch: 'main',
+    },
+    [...]
+  }
+}
+```
+
+### Ignored regions
+
+#### Component-based ignored region
+
+In case you need to ignore some components when running your tests, Sauce Labs Visual provides a way to ignore a list of components.
+
+An ignored component can be a specific element from the page.
+
+Those ignored components are specified when requesting a new snapshot.
+
+Example:
+
+```javascript
+    cy.sauceVisualCheck('login-page', {
+      ignoredRegions: [
+        cy.get('[data-test="username"]'),
+      ]
+    });
+```
+
+#### User-specified ignored region
+
+Alternatively, ignored regions can be user-specified areas. A region is defined by four elements.
+
+- `x`, `y`: The location of the top-left corner of the ignored region
+- `width`: The width of the region to ignore
+- `height`: The heigh of the region to ignore
+
+_Note: all values are pixels_
+
+Example:
+
+```javascript
+    cy.sauceVisualCheck('login-page', {
+      ignoredRegions: [
+        {
+          x: 240,
+          y: 800,
+          width: 1520,
+          height: 408
+        }
+      ],
+    });
+```
+
+### Specifying options for Cypress Screenshot
+
+Sauce Visual is relying on native screenshot feature from Cypress. As `cy.snapshot` has options, they can be specified in the `cy.visualCheck` command.
```

Comment:
As `cy.snapshot` has options, they can be specified in the `cy.visualCheck` command.

I'm not fully sure what we want to say here
```suggestion
Sauce Visual relies on the native screenshot feature from Cypress. Options in cy.snapshot can be specified in the cy.visualCheck command.
```